locators/patient-master.ts

```diff
@@ -6,6 +6,13 @@ export class PatientMaster {
   readonly closePatientMaster: Locator;
   readonly searchField: Locator;
   readonly table: Locator;
+  readonly heading: Locator;
+  readonly search: Locator;
+  readonly sear_result: Locator;
+  search_key_recieved: string;
```

Comment:
make this readonly

Reply:
search_key_recieved would change according to user input - if its readonly cannot assign it to changing values